### DISCUSSION TITLE
Custom themes don't work when cache is enabled

### DISCUSSION THREAD

**Discussion by rtest12** [keycloak/keycloak#8778] [Q&A]
I am creating a theme, everything works great if the cache is turned off. As soon as I turn it on, the changes disappear ..
I also tried making a theme in the archive jar (/standalone/deployments), but it does not help.
Speech specifically about the clause <cacheThemes> true </cacheThemes>
I saw that I was not the only one faced with this, but I could not find a solution in the search.

**Comment by laskasn** [comment]
We also have issues with the theme caching (see [here](https://github.com/keycloak/keycloak/discussions/8628)), but in a different scenario, which however, might occur due to the same underlying reason.
